build.py

```diff
@@ -382,13 +382,11 @@ def pytorch_cmake_args(images):
 
 def onnxruntime_cmake_args(images, library_paths):
     cargs = [
+        '-DTRITON_ENABLE_ONNXRUNTIME_TENSORRT=ON',
```

Comment:
We should enable ORT for Jetpack in a subsequent PR since it will manually building ORT on jetson first to unblock the CI.